docs/manifests/cloudprofile-kubevirt.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: core.gardener.cloud/v1beta1
+kind: CloudProfile
+metadata:
+  name: kubevirt
+spec:
+  type: kubevirt
+  providerConfig:
+    apiVersion: kubevirt.provider.extensions.gardener.cloud/v1alpha1
+    kind: CloudProfileConfig
+    machineDeploymentConfig:
+    - machineTypeName: "standard-1"
+      namespace: "default"
+    - machineTypeName: "standard-2"
+      namespace: "default"
+    machineImages:
+    - name: ubuntu
+      versions:
+      - version: "16.04"
+        sourceURL: "https://cloud-images.ubuntu.com/xenial/current/xenial-server-cloudimg-amd64-disk1.img"
+      - version: "18.04"
+        sourceURL: "https://cloud-images.ubuntu.com/bionic/current/bionic-server-cloudimg-amd64.img"
+  kubernetes:
+    versions:
+    - version: "1.17.5"
+  machineImages:
+  - name: "ubuntu"
+    versions:
+    - version: "16.04"
```

Comment:
one small nit, I would advise to use 16.10 instead of 16.04 because of the dns issues we were facing (systemd-resolved is default from 16.10 version). I didn't push my fix to os-ubuntu-extension yet, however, when I have some free time I would push it.

Reply:
OK, but could you then suggest an image URL for 16.10 that works?

Reply:
Also, after taking a look at https://cloud-images.ubuntu.com/, wouldn't it be better to use e.g. https://cloud-images.ubuntu.com/releases/xenial/release/ubuntu-16.04-server-cloudimg-amd64-disk1.img (and corresponding for 16.10, 18.04, etc.) instead of https://cloud-images.ubuntu.com/xenial/current/xenial-server-cloudimg-amd64-disk1.img?
As far as I understand, `current` is a daily build, while `releases` are released periodically with certain quality.

Reply:
so as we spoke on slack channel, let's remove 16.04 and keep the https://cloud-images.ubuntu.com/releases/bionic/release/ubuntu-18.04-server-cloudimg-amd64.img for 18.04

Reply:
Good suggestion. The `os-ubuntu` was developed and validated only with ubuntu 18.04.

Reply:
Removed 16.04 as suggested